fissa/core.py

```diff
@@ -296,7 +304,8 @@ def load(self, path=None):
                     continue
                 self.load(fullfname)
             return
-        print("Reloading data from cache {}...".format(path))
+
+        if self.verbosity > 0: print("Reloading data from cache {}...".format(path))
```

Comment:
It is much easier to tell when these print statements are activated if you use `>= threshold_where_it_turns_on` instead of `> threshold - 1`
```suggestion
        if self.verbosity >= 1:
            print("Reloading data from cache {}...".format(path))
```

Reply:
You're right, I'll implement that everywhere.